### PR TITLE
Grid Component

### PR DESCRIPTION
**Pull request by Tanay-J**
This PR adds Simple Grid component to ink UI component library.

### Other files can be ignored as they either include commits from previous PRs or minor code refactoring

Files to be reviewed (2)
- grid.html
- grid.css

**Demo**
image

### CODE REVIEW

**Review by OmKakatkar — APPROVED** [review]
Looks good to merge
Like the UI of the Component Library

**Review by Veekshith-Thoram — APPROVED** [review]
The code looks fine and good to mention the files to be reviewed.

File: docs/grid.html [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+                  <img src="https://picsum.photos/200" />
+                  <p class="text-center">Column 4</p>
+                </div>
+              </div>
+            </div>
+            <div class="code-container">
+              <iframe
+                src="https://www.thiscodeworks.com/embed/623e12882ef8dc0015ef5e7d"
+                style="width: 100%; height: 320px"
+                frameborder="0"
```
Comment (RahullRawat) [inline docs/grid.html:146]: Try to avoid inline styles

**Review by RahullRawat — APPROVED** [review]
- Your UI looks great
- It's simple with minimal use of color and contrast ratio is also good
- Good that you have mentioned which file to be reviewed
- One thing i want to suggest, try to provide preview link also
- Your naming convention is also good
- It's good that you have created so many useful utilities, so because of that you don't have to write too much css for grid.css

File: docs/grid.html (hunk 1 above)
Comment (Tanay-J) [reply docs/grid.html:146]: Thank you for elaborate feedback. The styling is provided by HTML snippet provider. I have kept it untouched.

**Pull request merged**